package.json

```diff
@@ -14,6 +14,7 @@
     "prebuild": "mkdir -p ./app/.tmp && cp ./app/src/app.config.json.tpl ./app/.tmp/app.config.json",
     "build": "ng build content-ce",
     "build.release": "npm run build -- --configuration=production,release",
+    "build-libs": "ng build aca-shared && ng build aca-folder-rules && ng build aca-content && ng build aca-about && ng build aca-settings && ng build aca-testing-shared && ng build adf-office-services-ext ",
```

Comment:
I think we need this just because the project does not use nx?

Reply:
yes, also I used it for my testing, just be sure that I did not break anything